qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/10_basic.yaml

```diff
@@ -83,6 +86,8 @@
   - match: { "acknowledged": true }
 
   - do:
+      warnings:
+        - "The stored search template API is deprecated. Use stored scripts instead."
```

Comment:
Oh boy. I think this will only emit this warning on the most recent version. `gradle check` will only run against that version but I think `gradle bwcTest` will fail with this. I *think* the right way to fix this is to only assert the warning on the appropriate version. But we don't have syntax for that. I think we should merge this as it because it passes `gradle check` and then fix the `gradle bwcTest` in a followup. I certainly could be wrong though.

Reply:
that sounds... fun :) Ill be watching CI regardless with fixes like this in mind :)